meson.build

```diff
@@ -64,31 +71,5 @@ if host_machine.system() != 'windows'
 endif
 thread_dep = dependency('threads')
 
-if host_machine.system() == 'linux'
-  libdrm_dep = dependency('libdrm', required : get_option('encoder'))
-  libvhal_dep = dependency('vhal-client',
-    required : get_option('encoder').enabled() or get_option('streamer').enabled())
-  libvpl_dep = dependency('vpl', required : get_option('encoder'))
-  libva_dep = dependency('libva', required : get_option('encoder'))
-
-  libavcodec_dep = dependency('libavcodec', required : get_option('encoder'))
-  libavfilter_dep = dependency('libavfilter', required : get_option('encoder'))
-  libavformat_dep = dependency('libavformat', required : get_option('encoder'))
-  libavutil_dep = dependency('libavutil', required : get_option('encoder'))
-
-  titansdk_dep = null_dep
-endif
-
-if host_machine.system() == 'windows'
-  winres_args = []
-  if build_machine.cpu_family() == 'x86_64'
-    winres_args += '-D_WIN64'
-  endif
-endif
-
-nlohmann_json_dep = dependency('nlohmann_json', required : get_option('streamer'))
-
-sioclient_dep = dependency('sioclient', required : get_option('streamer'))
-
 subdir('prebuilt')
```

Comment:
Original commits added a few lines here for ``windows -> build-deps`` in commit https://github.com/intel/cloud-streaming/pull/55/commits/20c6e270c5b24ae84bcce6987240dfc0c6d05303 and then removed them in https://github.com/intel/cloud-streaming/pull/55/commits/13a987b5b9736c94b3299bf294a2aff2e35d5035. Just noting the difference. Doesnt look like an issue.

Reply:
That's actually was the same block which you ask about in next https://github.com/intel/cloud-streaming/pull/55#discussion_r1378956652. Since I removed it from prebuilt/meson.build, there is no sense to leave it here. So, it's gone.